test_scripts/Resumption/Handling_errors_from_HMI/012_unsubscribe_from_the_same_vehicledata_without_request_to_hmi_by_error_in_sub_resultCode_during_resumption.lua

```diff
@@ -0,0 +1,138 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+--
+-- Description:
+-- In case:
+-- 1. App1 is subscribed to data_1, data_2 and data_3
+-- 2. App2 is subscribed to data_3 and data_4
+-- 3. Unexpected disconnect and reconnect are performed
+-- 4. App1 and app2 re-register with actual HashId
+-- SDL does:
+--  - start resumption process for both apps
+--  - send VI.SubscribeVehicleData(data_1, data_2, data_3) request related to app1 to HMI
+--  - not send VI.SubscribeVehicleData(data_3, data_4) request related to app2 to HMI
+-- 5. HMI responds with <erroneous> internal resultCode for data_1 to VI.SubscribeVehicleData(data_1, data_2, data_3) request
+-- SDL does:
+--  - process response from HMI
+--  - not restore subscription for app1 (data_1, data_2, data_3)
+--  - respond RegisterAppInterfaceResponse(success=true,result_code=RESUME_FAILED) to mobile application app1
+--  - send VI.UnsubscribeVehicleData(data_2) request related to app1 to HMI
+--  - send VI.SubscribeVehicleData(data_4) request related to app2 to HMI
```

Comment:
Is data_3 supposed to be included in this SubscribeVehicleData for app 2?

Reply:
SDL doesn't include `data_3` into `UnsubscribeVehicleData` request to HMI since it's shared data for both apps.
Hence SDL shouldn't include it into `SubscribeVehicleData` request for 2nd app. 
Subscription on `data_3` has already successfully established previously in scope of 1st app.